src/sfctl/custom_app.py

```diff
@@ -65,6 +68,167 @@ def validate_app_path(app_path):
             'Invalid path to application directory: {0}'.format(abspath)
         )
 
+def print_progress(current, total, rel_file_path, show_progress):
+    """Display progress for uploading"""
+    if show_progress:
+        print('[{}/{}] files, {}'.format(current, total,
+                                         rel_file_path),
+              file=sys.stderr)
+
+def parse_imagestore_connection_string(cluster_manifest_content): #pylint: disable=invalid-name
```

Comment:
Why invalid name here? Can you use a shorter name and remove the pylint disable?